controllers/article-controller.js

```diff
@@ -21,4 +26,19 @@ function getArticles(req, res, next) {
     });
 }
 
-module.exports = { getArticleById, getArticles };
+function patchArticle(req, res, next) {
+  const { inc_votes } = req.body;
+  const { article_id } = req.params;
+  Promise.all([
+    insertArticle(inc_votes, article_id),
+    checkArticleExists(article_id),
```

Comment:
Is this check necessary? Will psql gives us an error if it can't find the article_id? If it does, then it won't be necessary to make two reads to the db...